mentat/edit_history.py

```diff
@@ -53,14 +51,15 @@ async def redo(self) -> Optional[str]:
         for edit in edits_to_redo:
             edit.display_full_edit(code_file_manager.file_lines[edit.file_path])
         await code_file_manager.write_changes_to_files(edits_to_redo)
+        return []
 
-    def undo_all(self) -> str:
+    def undo_all(self) -> List[str]:
         if not self.edits:
-            return colored("No edits available to undo", color="light_red")
+            return ["No edits available to undo"]
 
         errors = list[str]()
         while self.edits:
             error = self.undo()
             if error:
-                errors.append(error)
-        return "\n".join(errors)
+                errors += error
```

Comment:
Should this still be `.append` (since it's a list)?

Reply:
error is also a list